WatchDogEclipsePlugin/WatchDog/src/nl/tudelft/watchdog/eclipse/logic/breakpoint/BreakpointClassifier.java

```diff
@@ -0,0 +1,38 @@
+package nl.tudelft.watchdog.eclipse.logic.breakpoint;
+
+import org.eclipse.debug.core.model.IBreakpoint;
+
+import nl.tudelft.watchdog.core.logic.breakpoint.BreakpointType;
+
+/**
+ * Estimates the nature of a breakpoint into one of {@link BreakpointType}.
+ */
+public class BreakpointClassifier {
+
+	/**
+	 * Classifies the breakpoint type of the given breakpoint by analyzing its
+	 * class name.
+	 */
+	public static BreakpointType classify(IBreakpoint breakpoint) {
+		switch (breakpoint.getClass().getSimpleName()) {
```

Comment:
Might there be a way that do not have to rely on a possibly changing name, but on types instead, such that the compiler would catch this error for us? :)